emscripten.py

```diff
@@ -1313,6 +1313,14 @@ def emscript_wasm_backend(infile, settings, outfile, libraries=None, compiler_en
     backend_args += ['-thread-model=single'] # no threads support in backend, tell llc to not emit atomics
     # disable slow and relatively unimportant optimization passes
     backend_args += ['-combiner-alias-analysis=false', '-combiner-global-alias-analysis=false']
+
+    # asm.js-style exception handling
+    if settings['DISABLE_EXCEPTION_CATCHING'] != 1:
+      backend_args += ['-enable-emscripten-cxx-exceptions']
+    if settings['DISABLE_EXCEPTION_CATCHING'] == 2:
```

Comment:
I see. Yeah, let's just fix both of them to use something with a double-underscore prefix.